src/routes/tools_routes.js

```diff
@@ -267,7 +295,7 @@ router.get('/cardimage/:id', async (req, res) => {
   try {
     let { id } = req.params;
 
-    const defaultPrinting = req?.query?.defaultPrinting;
+    const defaultPrinting = req?.query?.defaultPrinting || req?.user?.defaultPrinting;
```

Comment:
In previous fix we pass defaultPrinting from the frontend to the cardimage endpoint when inside the cube context. But cardimage is also used outside of cubes, so adding the user setting as the next fallback